oop1-construct.js

```diff
@@ -1,16 +1,62 @@
-//1. Create a Class called WhyClass.  This class should have one property, and two methods.  THe property is called "purpose", and stores a string.  This string should clearly explain why classes are used in Object Oriented Programming.  The first method is called explain().  This method should print the contents stored in the purpose 'field' of the class.  The second method is called pieces().  This 2nd method should print out the various parts that are essential to building a class.  Instantiate the class, and invoke the method so the results are showing in the console once this file is run.
+//1. Create a Class called WhyClass.  This class should have one property, and two methods.  THe property is called "purpose", and stores a string.  This string should clearly explain why classes are used in Object Oriented Programming. 
+// The first method is called explain().  This method should print the contents stored in the purpose 'field' of the class.  The second method is called pieces().  This 2nd method should print out the various parts that are essential to building a class.  
+//Instantiate the class, and invoke the method so the results are showing in the console once this file is run.
 
 //your code here...
+class WhyClass {
+    constructor(purpose) {
+
+        this.purpose = purpose;
+    }
+    explain(purpose) {
+        console.log(purpose);
+    }
+    pieces() {
+        console.log('The important pieces of a class are the contructor, getters, setters, methods and attributes.')
+    }
+
 
 
+}
+
+let example = new WhyClass("a class is a blueprint for creating objects (a particular data structure), providing initial values for state (member variables or attributes), and implementations of behavior (member functions or methods). The user-defined objects are created using the class keyword.");
+
+example.explain(example.purpose);
+example.pieces();
```

Comment:
See the declaration above to refactor this method. I'd put the message in a variable so it's easy to read and change in the future if needed. 

`const message = 'The important pieces of a class are the contructor, getters, setters, methods and attributes.';
example.pieces(message);`